Directory.Build.props

```diff
@@ -32,5 +32,10 @@
     <EnableDefaultEmbeddedResourceItems>false</EnableDefaultEmbeddedResourceItems>
   </PropertyGroup>
   
+  <!-- Common NuGet package properties -->
+  <PropertyGroup>
+    <PackageVersion>0.0.1-alpha</PackageVersion>
```

Comment:
Can you make this conditional on the property currently being empty? That way it can be overridden on the command line which would be useful for automated build scripts. Without this change, would setting it on the command line with `/p:PackageVersion=0.0.1-alpha` work today?

Reply:
Yes, I should have made it conditional on it not already being set. Fixed, and also verified `dotnet pack -p:PackageVersion=0.0.2` works as expected.